iceoryx_posh/source/roudi/service_registry.cpp

```diff
@@ -17,87 +17,83 @@
 
 #include "iceoryx_posh/internal/roudi/service_registry.hpp"
```

Comment:
Remark for reviewers:
The diff is almost useless, all implementations where changed, i.e. nothing except the interfaces remain.